client.py

```diff
@@ -149,9 +148,9 @@ async def list_sales(message):
             formatted_discounted_price = discounted_price.replace('CDN$ ','$')
             discounted_percent = games[key]['price_overview']['discount_percent']
             if discounted_percent > 0:
-                formatted_sales += f'\n•\t**{game_title}** is on sale for {formatted_discounted_price} - {discounted_percent}% off!\n\t  {game_url}'
+                formatted_sales += f'\n•\t**{game_title.title()}** is on sale for {formatted_discounted_price} - {discounted_percent}% off!\n\t  {game_url}'
     else:
-        formatted_sales = f'{line_string}\nNo games from your wishlist are currently on sale :sob:'
+        formatted_sales = f'{line_string}\nNo games from your wishlist are currently on sale.'

```

Comment:
Reasonable place to sob if you wanna keep it in

Reply:
would like to sob